lib/util/middleware.js

```diff
@@ -2,6 +2,7 @@ if (typeof (fetch) === 'undefined') require('isomorphic-fetch')
 
 const API_USER_PATH = '/api/secure/user'
 const API_MONITORTRIP_PATH = '/api/secure/monitoredtrip'
+const API_ITINERARYCHECK_PATH = '/api/secure/itinerarycheck'
```

Comment:
A general comment for these constants: please insert missing underscores to make them human readable.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/ea97ce635402164e163c1f46be35c6ac7d971697.